### PR TITLE
[LoongArch] Support bswap for LSX/LASX VTs

### PR DESCRIPTION
**Pull request by xen0n**
On top of #114170

### CODE REVIEW

File: llvm/lib/Target/LoongArch/LoongArchISelLowering.cpp [hunk 1]
```diff
@@ -269,6 +269,9 @@ LoongArchTargetLowering::LoongArchTargetLowering(const TargetMachine &TM,
           {ISD::SETNE, ISD::SETGE, ISD::SETGT, ISD::SETUGE, ISD::SETUGT}, VT,
           Expand);
     }
+    for (MVT VT : {MVT::v8i16, MVT::v4i32, MVT::v2i64}) {
+      setOperationAction(ISD::BSWAP, VT, Legal);
+    }
```
Comment (heiher) [inline llvm/lib/Target/LoongArch/LoongArchISelLowering.cpp:274]: Remove braces.

https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

File: llvm/lib/Target/LoongArch/LoongArchISelLowering.cpp [hunk 2]
```diff
@@ -317,6 +320,9 @@ LoongArchTargetLowering::LoongArchTargetLowering(const TargetMachine &TM,
           {ISD::SETNE, ISD::SETGE, ISD::SETGT, ISD::SETUGE, ISD::SETUGT}, VT,
           Expand);
     }
+    for (MVT VT : {MVT::v16i16, MVT::v8i32, MVT::v4i64}) {
+      setOperationAction(ISD::BSWAP, VT, Legal);
+    }
```
Comment (heiher) [inline llvm/lib/Target/LoongArch/LoongArchISelLowering.cpp:325]: Ditto.

**Review by heiher — APPROVED** [review]
LGTM with nits. Thanks.

File: llvm/lib/Target/LoongArch/LoongArchISelLowering.cpp (hunk 2 above)
Comment (xen0n) [reply llvm/lib/Target/LoongArch/LoongArchISelLowering.cpp]: Done.

**Review by SixWeining — APPROVED** [review]
LGTM. Thanks.

**Comment by SixWeining** [comment]
need rebase since #114170 is merged

**Pull request merged**